### DISCUSSION TITLE
buildkit pod timeout

### DISCUSSION THREAD

**Discussion by varun-vijayagopal** [docker/buildx#2277] [General]
We use buildx and buildkit to build images on our kubernetes cluster. At times the buildkit pod ( driver ) takes more than 2 mins to come up and this causes buildx build to fail with below 

#0 building with "docker-listing-916" instance using kubernetes driver

#1 [internal] booting buildkit
#1 waiting for 1 pods to be ready
#1 waiting for 1 pods to be ready 110.0s done
#1 ERROR: expected 1 replicas to be ready, got 0

Is there a way to increase timeout or wait time for this ?

**Comment by ghost** [comment]
According to the source code, this is hardcoded and cannot be customized. I suggest that this be fixed because it is not appropriate for this to be non-configurable.

https://github.com/docker/buildx/blob/1f28985d2014a3e9e5e5fc05480b315ef53c67fb/driver/kubernetes/driver.go#L98-L120
